core/primitives/src/trie_key.rs

```diff
@@ -338,6 +361,15 @@ impl TrieKey {
                 buf.extend(&index.to_le_bytes());
             }
             TrieKey::BandwidthSchedulerState => buf.push(col::BANDWIDTH_SCHEDULER_STATE),
+            TrieKey::BufferedReceiptGroupsQueueData { receiving_shard: receiver_shard } => {
+                buf.push(col::BUFFERED_RECEIPT_GROUPS_QUEUE_DATA);
+                buf.extend(&receiver_shard.to_le_bytes());
```

Comment:
Oh I think the renaming might not have worked here and below, could you please change `receiver_shard` to `receiving_shard`?

Also, could we do something like above where we convert shard_id to u16 first?

Reply:
> Oh I think the renaming might not have worked here and below, could you please change receiver_shard to receiving_shard?

:+1:, fixed. I used rust-analyzer to rename the variable and it didn't work properly here.

> Also, could we do something like above where we convert shard_id to u16 first?

I don't like the idea of converting the shard id from `u64` to `u16` in TrieKey. It makes sense in bandwidth requests where we care about optimizing space, but for trie keys I'd prefer to prioritize being future proof. I don't think we gain much by optimizing it to u16 there.